builds/delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -90,6 +95,26 @@ node {
 		if (!service_config) {
 			error "Failed to fetch service metadata from catalog"
 		}
+		// Get the Account Details from the ConfigLoader
+		accountInfo = utilModule.getAccountInfo(service_config)
+
+    if(!service_config.region) {
```

Comment:
Fix formatting..

Reply:
the formatting is correct in the file. Not sure why is it showing different in the diff